src/FakeItEasy/Creation/IFakeAndDummyManager.cs

```diff
@@ -13,7 +13,7 @@ internal interface IFakeAndDummyManager
         /// <param name="typeOfDummy">The type of dummy to create.</param>
         /// <returns>The created dummy.</returns>
         /// <exception cref="FakeItEasy.Core.FakeCreationException">The current IProxyGenerator is not able to generate a fake of the specified type and
-        /// the current IFakeObjectContainer does not contain the specified type.</exception>
+        /// the current IDummyValueCreationSession does not contain the specified type.</exception>

```

Comment:
Thanks. The only thing I'd be careful of is that in our case, it's not necessarily the proxy generator that can't make a Dummy. Well, it can't, but neither can any of our other resolution strategies; I wouldn't focus just on the proxy generator.